app/routes/meetings/index.tsx

```diff
@@ -1,17 +1,17 @@
-import { Route, Routes } from 'react-router-dom';
+import loadable from '@loadable/component';
 import PageNotFound from '../pageNotFound';
-import MeetingDetailWrapper from './MeetingDetailWrapper';
-import MeetingEditor from './components/MeetingEditor';
-import MeetingList from './components/MeetingList';
+import type { RouteObject } from 'react-router-dom';
 
-const MeetingRoute = () => (
-  <Routes>
-    <Route index element={<MeetingList />} />
-    <Route path="create" element={<MeetingEditor />} />
-    <Route path=":meetingId" element={<MeetingDetailWrapper />} />
-    <Route path=":meetingId/edit" element={<MeetingEditor />} />
-    <Route path="*" element={<PageNotFound />} />
-  </Routes>
-);
+const MeetingList = loadable(() => import('./components/MeetingList'));
+const MeetingEditor = loadable(() => import('./components/MeetingEditor'));
+const MeetingDetailWrapper = loadable(() => import('./MeetingDetailWrapper'));
 
-export default MeetingRoute;
+const MeetingsRoute: RouteObject[] = [
```

Comment:
```suggestion
const meetingsRoute: RouteObject[] = [
```
I think all the routes should start with a lower case letter, as they are not react components.